bedrock/teach/templates/teach/smarton/surveillance.html

```diff
@@ -0,0 +1,368 @@
+              </ul>
+            </div>
+          </div>
+        </div>
+
+        <div class="tip">
+          <h4>{{ _('Device encryption') }}</h4>
+          <p>{{ _('Use these on your devices to make sure the only eyes that see your personal information are the ones you authorize.') }}</p>
+
+          <div class="columns columns-2">
+            <div class="col">
+              <h5>{{ _('Mac') }}</h5>
+              <p>{{ _('<a href="{url}">FileVault</a> — a free and built-in way to encrypt your Mac’s startup disk').format(url='https://support.apple.com/HT204837')|safe }}</p>
+
+              <h5>{{ _('Windows') }}</h5>
+              <p>
+                {{ _('Use <a href="{url1}">BitLocker</a> (Windows Pro) or a free program called <a href="{url2}">Diskcryptor</a> (standard Windows versions).').format(url1='http://windows.microsoft.com/windows-8/bitlocker-drive-encryption',url2='https://diskcryptor.net/wiki/Main_Page')|safe }}
+              </p>
+
+              <h5>{{ _('Linux') }}</h5>
+              <p>{{ _('Full-disk encryption is usually offered when you set up your system (it’s called LUKS).') }}</p>
+            </div>
+
+            <div class="col">
+              <h5>{{ _('Android') }}</h5>
+              <p>{{ _('<a href="{url}">See instructions here</a> for your Android device.').format(url='https://support.google.com/nexus/answer/2844831?vid=1-635761320342085866-3234499977')|safe }}</p>
+
+              <h5>{{ _('iOS') }}</h5>
+              <p>{{ _('Have an iPhone, iPad or iPod? <a href="{url}">See instructions here</a>.').format(url='https://support.apple.com/HT202064')|safe }}</p>
+            </div>
+          </div>
+        </div>
+
+      </div>
+
+    </div>
+  </section>
+
+  <section id="chat" class="section section-chat">
+    <div class="content">
+
+      <h2 class="section-title">{{ _('Blast your surveillance smarts') }}</h2>
+      <p class="section-intro">
+        {{ _('Now that you have tools to help protect yourself, spread the word to friends.') }}
+        {{ _('The only way we’ll get a handle on this fundamental issue of technology is if we keep the conversation going.') }}
+      </p>
+
+      <ol class="big-list with-sidebar">
+        <li>
+          <p>
+            {# Hack: needed to shorten the string for English; translations were already short enough. #}
+            {% set q1tweet = 'Which would you be more comfortable sharing with the government: your bank account or your DNA profile?' if LANG.startswith('en') else _('Which would you be more comfortable sharing with the government: your checking account number or your DNA profile?') %}
+
+            {{ q1tweet }}
+            <span class="tweet-this">
+              <a class="button-tweet" href="{{ 'https://www.twitter.com/intent/tweet?url=%s&amp;text=%s%s'|format('http://mzl.la/surveillance'|urlencode, q1tweet|urlencode, hashtag|urlencode)|e }}">{{ _('Tweet this') }}</a>
```

Comment:
Looks like the HTML entity plus urlencode is redundant, just a plain & with urlencode should do it.